### PR TITLE
feat(onboarding): Add initial onboarding changes

### PR DESCRIPTION
**Pull request by rsanchezbalo**

### CODE REVIEW

File: app/partials/common/lightbox-feedback.jade [hunk 1]
```diff
@@ -1,8 +1,23 @@
...
-        button.button-green.button-large.button-block(type="submit", title="{{'LIGHTBOX.FEEDBACK.ACTION_SEND' | translate}}", translate="LIGHTBOX.FEEDBACK.ACTION_SEND")
+        textarea(
+            ng-model="feedback.comment",
+            data-required="true"
+        )
+    fieldset.actions
+        button.btn-big(
+            variant="primary"
+            type="submit",
+            title="{{'LIGHTBOX.FEEDBACK.ACTION_SEND' | translate}}", translate="LIGHTBOX.FEEDBACK.ACTION_SEND"
```
Comment (Xaviju) [inline app/partials/common/lightbox-feedback.jade:14]: Why single line here?

File: app/partials/common/lightbox-feedback.jade [hunk 2]
```diff
@@ -1,8 +1,23 @@
...
+        )
+    .can-also
+        h3 {{'LIGHTBOX.FEEDBACK.YOU_CAN_ALSO' | translate}}
+        .text
+            span {{'LIGHTBOX.FEEDBACK.CREATE' | translate}}
+            a(href="#") {{'LIGHTBOX.FEEDBACK.PUBLIC' | translate}}
```
Comment (Xaviju) [inline app/partials/common/lightbox-feedback.jade:20]: ?? `a(href="#")`

**Review by Xaviju — CHANGES_REQUESTED** [review]

File: app/partials/common/lightbox-feedback.jade (hunk 2 above)
Comment (rsanchezbalo) [reply app/partials/common/lightbox-feedback.jade:20]: Asked about this, UX told me to leave it like that.

File: app/partials/common/lightbox-feedback.jade (hunk 1 above)
Comment (rsanchezbalo) [reply app/partials/common/lightbox-feedback.jade:14]: My text editor pranked me, il change it.

**Pull request merged**